sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetRelation.scala

```diff
@@ -136,14 +125,7 @@ private[sql] class DefaultSource
       sqlContext.conf.writeLegacyParquetFormat.toString)
 
     // Sets compression scheme
-    conf.set(
```

Comment:
the old nested map, getOrElse was super confusing...

@liancheng / @HyukjinKwon let's avoid doing things like this in the future